Assets/Mirror/Components/NetworkTransformBase.cs

```diff
@@ -153,6 +163,15 @@ protected virtual void Apply(TransformSnapshot interpolated)
             if (syncPosition) target.localPosition = interpolated.position;
             if (syncRotation) target.localRotation = interpolated.rotation;
             if (syncScale)    target.localScale = interpolated.scale;
+
+            if (syncPosition)
+                target.localPosition = interpolatePosition ? interpolated.position : endGoal.position;
```

Comment:
what about the 'if' cases above?
they already set .localposition, and here we do it again?
shouldn't it be merged?

Reply:
Good catch, looks like i forgot to remove the older 3 lines <face palming monkey>